core-groovy/src/test/groovy/com/baeldung/closures/ClosuresUnitTest.groovy

```diff
@@ -0,0 +1,110 @@
+package com.baeldung.closures
+
+import spock.lang.Specification
+
+class ClosuresUnitTest extends GroovyTestCase {
+
+    Closures closures = new Closures()
+    
+    void testClosures() {
```

Comment:
Can we have separate tests for each construct with names describing each use case.

Reply:
Done